### DISCUSSION TITLE
NuGet package

### DISCUSSION THREAD

**Discussion by minexew** [pleonex/xdelta-sharp#5] [Q&A] [answered]
Hi, any chance for a NuGet release? :)

We would like to use XdeltaSharp for [Gnoll](https://github.com/Nefaro/gnoll)'s mod installer.

**Comment by pleonex** [comment]
Hi!

Yes! That would be great. I just need to finish a change to rename the namespace and improve the continuous integration and then I will push a NuGet :)

**Reply by pleonex** [reply]
I have just realized there is already a preview nuget in this feed:
https://dev.azure.com/pleonex/Pleosoft/_packaging?_a=feed&feed=Pleosoft-Preview

To use this preview NuGet you need to create a `nuget.config` file in the same directory as the `.sln` file of your project with this content:
```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <add key="Pleosoft-Preview" value="https://pkgs.dev.azure.com/pleonex/Pleosoft/_packaging/Pleosoft-Preview/nuget/v3/index.json" />
  </packageSources>
</configuration>
```

**Comment by pleonex** [comment, marked as answer]
Package published! https://www.nuget.org/packages/PleOps.XdeltaSharp
